pkg/smclient/client.go

```diff
@@ -163,22 +165,32 @@ func (client *serviceManagerClient) RegisterBroker(broker *types.Broker) (*types
 	return newBroker, nil
 }
 
-// ListBrokers returns brokers registered in the Service Manager
-func (client *serviceManagerClient) ListBrokers() (*types.Brokers, error) {
+// ListBrokersWithQuery returns brokers registered in the Service Manager satisfying provided queries
+func (client *serviceManagerClient) ListBrokersWithQuery(fieldQuery string, labelQuery string) (*types.Brokers, error) {
 	brokers := &types.Brokers{}
-	err := client.list(brokers, "/v1/service_brokers")
+	err := client.list(brokers, "/v1/service_brokers?fieldQuery="+fieldQuery+"&labelQuery="+labelQuery)
 
 	return brokers, err
 }
 
-// ListPlatforms returns platforms registered in the Service Manager
-func (client *serviceManagerClient) ListPlatforms() (*types.Platforms, error) {
+// ListBrokers returns brokers registered in the Service Manager
+func (client *serviceManagerClient) ListBrokers() (*types.Brokers, error) {
+	return client.ListBrokersWithQuery("", "")
```

Comment:
Is there a test which checks whether this method works? With empty query parameters?